requirements.txt

```diff
@@ -0,0 +1,39 @@
+certifi==2020.12.5
+chardet==4.0.0
+click==7.1.2
+filelock==3.0.12
+flask==2.0.2
+idna==2.10
+itsdangerous==1.1.0
+Jinja2==2.11.3
+joblib==1.0.1
+Keras-Preprocessing==1.1.2
+MarkupSafe==1.1.1
+matplotlib==3.4.3
+numpy==1.20.2
+opencv-python==4.5.1.48
+packaging==20.9
+pandas==1.3.4
+Pillow==8.2.0
+pyparsing==2.4.7
+pytesseract==0.3.7
+python-dateutil==2.8.1
+pytz==2021.1
+regex==2021.4.4
+requests==2.25.1
+sacremoses==0.0.45
+scikit-learn==0.24.2
+scipy==1.6.3
+sentencepiece==0.1.95
+six==1.15.0
+sklearn==0.0
+threadpoolctl==2.1.0
+tokenizers==0.10.3
+torch==1.8.1
+torchvision==0.9.1
+tqdm==4.50.0
+transformers==4.8.2
```

Comment:
I updated the transformers and tokenizers version, the old codes used the transformers version of 3.2.0.